scripts/create_test_subset.py

```diff
@@ -105,9 +114,18 @@ def main(
     if families_n is not None:
         fams = fapi.get_families(project)
         all_families = [family['id'] for family in fams]
-        families = random.sample(all_families, families_n)
-        pedigree = fapi.get_pedigree(project=project, internal_family_ids=families)
+        full_pedigree = fapi.get_pedigree(
+            project=project, internal_family_ids=all_families
+        )
+        external_family_ids = _get_random_families(full_pedigree, families_n)
```

Comment:
You mentioned you ended up selecting families manually. Perhaps we could use an option to provide family IDs as input to the script?

Reply:
Or with `_get_random_families` the selection ends up better?

Reply:
Yeah so _get_random_families just does what I was doing with that manual selection :)